package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "hexo-filter-nofollow",
+  "version": "2.0.0",
+  "description": "Add nofollow attribute to all external links automatically",
+  "main": "index.js",
+  "directories": {
+    "lib": "./lib"
+  },
+  "files": [
+    "lib/",
+    "index.js"
+  ],
+  "engines": {
+    "node": ">= 8.6.0"
+  },
+  "scripts": {
+    "eslint": "eslint .",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
```

Comment:
can simply to `"repository": "hexojs/hexo-filter-nofollow",`. Hexo repos are using this simpler syntax.